docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,129 @@
 [[apm-open-telemetry]]
-== OpenTelemetry integration
+== Use OpenTelemetry with APM
 
-https://opentelemetry.io/docs/concepts/what-is-opentelemetry/[OpenTelemetry] is a set of APIs, SDKs, tooling, and integrations that enable the capture and management of
-telemetry data from your services and applications. For more information about the
-OpenTelemetry project, see the https://github.com/open-telemetry/opentelemetry-specification/blob/master/README.md[spec].
+++++
+<titleabbrev>Use OpenTelemetry</titleabbrev>
+++++
 
-[float]
-== OpenTelemetry and the {stack}
+https://opentelemetry.io/docs/concepts/what-is-opentelemetry/[OpenTelemetry] is a set of APIs, SDKs, tooling,
+and integrations that enable the capture and management of telemetry data from your services and applications.
 
-[subs=attributes+]
-include::./diagrams/apm-otel-architecture.asciidoc[Architecture of Elastic APM with OpenTelemetry]
+// Improve wording?
+// TODO: Update link when landing page is ready
+[NOTE]
+====
+To learn more about using OpenTelemetry with Elastic _outside_ the context of APM,
+refer to https://github.com/elastic/opentelemetry[Use OpenTelemetry with Elastic].
+====
 
 Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation
 to easily send observability data to the {stack}.
 There are several ways to integrate OpenTelemetry with the {stack}:
 
-**OpenTelemetry API/SDK with Elastic APM agents**
+* <<apm-otel-elastic-distros,Elastic Distributions for OpenTelemetry>>
+* <<apm-otel-api-sdk,OpenTelemetry API/SDK>>
+* <<apm-otel-collector,OpenTelemetry Collector>>
+* <<apm-otel-api-sdk-elastic-agent,OpenTelemetry API/SDK + Elastic APM agent>>
+* <<apm-otel-lambda,Lambda collector exporter>>
+
+[float]
+[[apm-otel-elastic-distros]]
+== Elastic Distributions for OpenTelemetry
+
+Elastic offers several distributions for OpenTelemetry.
+A _distribution_ is a customized version of an upstream OpenTelemetry repository.
+Each Elastic Distribution for OpenTelemetry is an extension of an https://opentelemetry.io/docs/languages/[OpenTelemetry Language API or SDK].
+
+image::images/apm-otel-distro.png[]
+
+With an Elastic distro you have access to all the features of the OpenTelemetry API or SDK that it extends, plus:
```

Comment:
Would you want to call out the use of "distro" for distribution? Maybe for an OTel users it's understood, so it doesn't need to be.